app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "helpers.application_helper.text"
+
+    if page_title.empty?
```

Comment:
đưa đoạn if else này lên 1 dòng thôi

Reply:
Em chuyển về toán tử 3 ngôi con bot nó check lỗi a ạ!